test/common/markdown/package.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Maybe place the file in `test\tools\markdown`? It seems we planned to move all test helper tools in the `test\tools` directory.